packages/yoga/src/Popover/web/Popover.jsx

```diff
@@ -43,9 +43,17 @@ function Popover({
         </PopoverContainer>
       )}
 
-      <div onMouseEnter={handleShowPopover} onMouseLeave={handleHidePopover}>
+      <button
+        type="button"
+        style={{ all: 'unset' }}
```

Comment:
Tried to create a styled-component. But eslint throws this:
![Screenshot 2023-03-29 at 11 10 30](https://user-images.githubusercontent.com/796443/228501978-ca440810-3257-425e-87fc-01b02d758b27.png)

So, for simplicity i decided to keep the style (reset of it :D) inline.